CHANGELOG.md

```diff
@@ -26,7 +26,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Mark make_unique argument of create_target Gmpv8 as deprecated and ignore it.
   It is already ignored by gvmd with GMP 8 [#156](https://github.com/greenbone/python-gvm/pull/156)
 
-[unreleased]: https://github.com/greenbone/python-gvm/compare/v1.0.0...master
+[1.1.0]: https://github.com/greenbone/python-gvm/compare/v1.0.0...master
```

Comment:
also update the URL

Reply:
the diff is still correct. i'll update it when we are doing the relase and the tag has been created.